app/server/utils/intercom.py

```diff
@@ -2,8 +2,23 @@
 from flask import current_app
 
 
-def create_intercom_android_secret(user_id):
-    secret_bytes = current_app.config['INTERCOM_ANDROID_SECRET'].encode('ascii')
+def create_intercom_secret(user_id, device_type):
+    """
+    Function to generate an intercom secret using HMAC for identity verification for customer support.
+
+    :param user_id: Sempo UserID to generate a unique hash.
+    :param device_type: Web, Android or iOS
+    :return:
+    """
+    if device_type is 'WEB':
+        secret = current_app.config['INTERCOM_WEB_SECRET']
+    elif device_type is 'ANDROID':
+        secret = current_app.config['INTERCOM_ANDROID_SECRET']
+    else:
+        return Exception('No device_type provided')
```

Comment:
`return Exception(f'device_type {device_type} provided not recognised. Allowed device types are {ALLOWED_INTERCOM_DEVICE_TYPES}')